microsoft-edge/webview2/get-started/winforms.md

```diff
@@ -68,71 +82,98 @@ Use NuGet to add the WebView2 SDK to the project.
 
 Add a WebView to your app.  
 
-1.  Open the **Windows Forms Designer**.  
-1.  Search for **WebView2** in the **Toolbox**.  
+1. Choose **Project** > **Add Form (Windows Forms)...**.  
+1. In the Add New Item panel, choose **Visual C# Items** > **Web** > **Windows Forms** > **Form (Windows Forms)** and choose **Add**.
+1. Choose **View** > **Toolbox**.
+1. In the **Toolbox** choose **WebView2 Windows Forms Control** to expand the options.  
     
     > [!NOTE]
-    > If you are using Visual Studio 2017, by default **WebView2** may not display in the **Toolbox**.  To enable the behavior, choose **Tools** > **Options** > **General** > set the **Automatically Populate Toolbox** setting to `True`.  
+    > If you are using Visual Studio 2017, by default **WebView2** may not display in the **Toolbox**. To enable the behavior, choose **Tools** > **Options** > **General** > and set the **Automatically Populate Toolbox** setting to `True`.  
     
-    Drag and drop the **WebView2** control into the Windows Forms App.
+1. Drag and drop the **WebView2** control into the Windows Forms App.
     
-    :::image type="complex" source="./media/winforms-toolbox.png" alt-text="Toolbox displaying WebView2":::
+    :::image type="complex" source="./media/winforms-toolbox.png" alt-text="Toolbox displaying WebView2" lightbox="./media/winforms-toolbox.png":::
        Toolbox displaying WebView2  
     :::image-end:::  
     
-1.  Set the `(Name)` property to `webView`.
+1. Close the **Toolbox**.
+
+1. In the Properties panel, set the **(Name)** property to **webView**. Use the **Categorized** and **Alphabetical** sort options as needed to find properties.
     
-    :::image type="complex" source="./media/winforms-properties.png" alt-text="Properties of the WebView2 control":::
+    :::image type="complex" source="./media/winforms-properties.png" alt-text="Properties of the WebView2 control" lightbox="./media/winforms-properties.png":::
        Properties of the WebView2 control
     :::image-end:::
     
-1.  The `Source` property sets the initial URI displayed in the WebView2 control.  Set the `Source` property to `https://www.microsoft.com`.  
-    
-    :::image type="complex" source="./media/winforms-source.png" alt-text="The Source property of the WebView2 control":::
-       The **Source** property of the WebView2 control
-    :::image-end:::  
+1.  The **Source** property sets the initial URI displayed in the WebView2 control. Set the **Source** property to `https://www.microsoft.com`.
 
-To build and run your project, select `F5`.  Ensure your WebView2 control displays [https://www.microsoft.com][MicrosoftMain].
+1. Select **F5** to build and run your project.
```

Comment:
One philosophy of doc'ing UI is to give GUI instructions (verbose version: "Select the Foo menu and then select Blah.  or:  "Select Foo > Bar.")  The UI shows the shortcut keys so the docs don't need to mention shortcut keys.  Examples in MWSG:  https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui